test/integration/default/build/aws.tf

```diff
@@ -6,6 +6,8 @@ provider "aws" {
   version = "= 1.1"
 }
 
+provider "random" {}
```

Comment:
Prefer not using random.

Reply:
Currently we are using it as a way to make unique bucket names.  Is there a specific way to do that?
Thanks!